internal/cmd/usage_md_test.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/tetratelabs/func-e/internal/globals"
 )
 
-const siteMarkdownFile = "../../site/usage.md"
+const siteMarkdownFile = "../../USAGE.md"
```

Comment:
is this going to break the existing redirect from getenvoy.io?

Reply:
though it's not a problem at all😄

Reply:
yep it will and I'll fix it before anyone notices :D

Reply:
https://github.com/tetratelabs/getenvoy.io/pull/182